pkg/tracing/interceptors/client.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 Michal Witkowski. All Rights Reserved.
+// See LICENSE for licensing terms.
+
+// gRPC Prometheus monitoring interceptors for client-side gRPC.
+
+package interceptors
```

Comment:
Just want to double check, we will remove those vendored code after Thanos fully migrated to OTEL based tracing?

Reply:
Yes, thats the intention.